pom.xml

```diff
@@ -481,7 +487,7 @@
         <maven.buildnumber.plugin.version>1.4</maven.buildnumber.plugin.version>
         <org.apache.felix.annotations.version>1.2.4</org.apache.felix.annotations.version>
         <identity.governance.version>1.4.75</identity.governance.version>
-        <carbon.identity.framework.version>5.19.20</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.20.93</carbon.identity.framework.version>
```

Comment:
need to uodate the carbon identity framework version here after merging https://github.com/wso2/carbon-identity-framework/pull/3577